Orange/widgets/cpewidget.py

```diff
@@ -79,6 +89,15 @@ def property_changed(self, name):
                 else:
                     openchar, closechar = "{", "}"
                 content = getattr(self, name)
+                # parse list default value and return
+                if content in NoneUIValue:
+                    setattr(self.node, name, None)
```

Comment:
If you repeat this code here from the last phase of the func at the bottom then someone later might edit what happens in that phase and forget to also change this copy up here, so I wouldn't do that -- can you instead set content to "None" here and then skip all the below modifications to content (all the way down to right before the try/catch that evaluates content and assigns the result to value)? Then you don't have to duplicate code and break the sequence of of processing phases in this function.